src/main/java/Sb2.java

```diff
@@ -131,5 +132,24 @@ public int getScriptCountForSprite(String spriteName) {
     public int[] getScriptLengthsForSprite(String spriteName) {
         return sprites.getScriptLengthsForSprite(spriteName);
     }
-}
+    /**
+     * Gets the number of global variables within the stage object.
+     * @return numVariables The number of global variables
+     */
+    public int getGlobalVariableCount() {
```

Comment:
What do you mean computes global state? What would be an architecture you would prefer? I had considered creating a get variable method, but the simplest way to implement that would tie together Sb2 and sprites even further.